main.go

```diff
@@ -103,8 +103,17 @@ func parseRawDate(rawDate string) (int, int, int, bool) {
 	return mm, dd, yy, true
 }
 
+func showDateAD(w io.Writer, t time.Time) {
```

Comment:
Slight asymmetry here against the `showDateBS` function which does both the conversion and the writing whereas this one only does the writing but should be fine as this will get moved around in any case